src/installation/install_epinio_on_rancher.md

```diff
@@ -0,0 +1,56 @@
+# Rancher configuration
+
+Before installing Epinio, you need a running Rancher instance.
+
+If you don't get it installed yet, feel free to follow installation guides [here](https://rancher.com/docs).
+
+## Epinio Prerequisites
+
+### Kubernetes cluster
+The supported configuration is to either deploy a new cluster or import an existing one in Rancher (those steps are detailed in the Rancher docs).</br>
+*For development purpose, you can imagine deploying Epinio on the same node where Rancher is installed but it requires more steps and keep in mind it's not supported by Rancher.*
+
+### Rancher settings
+At the time of writing, Epinio is not included in the latest stable Rancher version.</br>
+This means you need to update two settings in your Rancher dashboard.</br>
+Go to `Global settings` > `Advanced settings`  and update the following settings:</br>
+
+| | |
+|--|--|
+| `ui-dashboard-index`  | `https://releases.rancher.com/dashboard/epinio-v0.3.0/index.html` |
+| `ui-offline-preferred`  | `Remote` |
+
+*Please, keep in mind that `ui-dashboard-index` value will change over the time until it is not merged in stable Rancher version.*
```

Comment:
I have trouble understanding this sentence. Does it mean that the setting will be removed from Rancher in some future release ?

Reply:
Yes exactly, Epinio's UI code is not included yet in the Rancher code.
However, maybe I can mention versions of all components I used at the beginning of the document.

Reply:
Ok. So I managed to pull the correct meaning out of this. While I am still convinced that it could be phrased better, I also have no idea how to rephrase it. Maybe @agracey can help ? Lets ignore this for now.